### DISCUSSION TITLE
Take snapshot (and link highlighted items)

### DISCUSSION THREAD

**Discussion by Potato-Tyrant** [iaincollins/icarus#11] [Ideas & Suggestions]
First of all, thank you, I really like this program!

What I would like even more would be a way to save a snapshot of the whole page (even better if with fixed resolution).

For smaller systems one could use the windows screenshot, but with the likes of Sol (really "long" systems) or Maia (really "tall" systems), its tricky and doesn't look nice.

Also, I think it would be neat to have the ability to link the highlighted item to other CMDRs (maybe as EDSM link?)

Thanks again!

**Comment by iaincollins** [comment]
Hi there!

That's really interesting, I hadn't thought of something like this.

If folks are interested in sharing systems, I'd be happy to consider making a web based version where folks could share interactive versions of the same system map directly, but linking to ESDM also seems like a great idea.

**Reply by alyyasser** [reply]
i think, a convenient share button that generate an image to the clipboard would be enough for starter.
